cfme/base/ui.py

```diff
@@ -39,20 +49,55 @@ class LoggedIn(CFMENavigateStep):
 
     def step(self):
         login_view = self.create_view(LoginPage)
-        login_view.log_in(self.obj.appliance.user)
```

Comment:
I need to check this out, your logic is sound, however I _think_ login will only login if it needs to log in. Hence it shuldn't keep logging in. This check needs to be made though in case the appliance.user was updated to a different user, ie if we are using a user context manager.